### PR TITLE
Feature/var define

### PR DESCRIPTION
**Pull request by IhorNehrutsa**
Some comments in code.

### CODE REVIEW

**Comment by prusnak** [comment]
Can you please also update `example/modexample.c` to see how the generated code looks like?

**Comment by IhorNehrutsa** [comment]
All CONSTANT_TYPES = (type(None), int, bool, float, str, tuple) supported as module constants and class constants in generated C code.
ESP32 port tested.

**Comment by IhorNehrutsa** [comment]
The correct name for this feature is 'module_and_class_constanst'.
I think it is ready for use.
The next feature will be 'function_params_and_return'.

**Comment by IhorNehrutsa** [comment]
Oh, I'm sorry. 
git-flow closed the PR.

File: example/example.py [hunk 1]
```diff
@@ -1,39 +1,90 @@
...
 
-def func_b(arg1):
+def func_b(arg1: int) -> None:
     pass
 
-def func_c(arg1, arg2) -> None:
-    pass
+def func_c(arg1: int, arg2: int) -> int:
+    res = arg1 + arg2 * arg2 
+    return res
```
Comment (prusnak) [inline example/example.py]: Do we want to keep this here? Other functions use `pass` to indicate this Python code does not get magically rewritten into C

**Comment by IhorNehrutsa** [comment]
Not today. But maybe we could get the source code from 'module.py'?

**Pull request closed without merging**